contrib_bots/lib/john.py

```diff
@@ -0,0 +1,115 @@
+# pip install chatterbot is necessary
+import os
+import json
+import time
+import threading
+from random import choice
+from chatterbot import ChatBot
+from chatterbot.trainers import ChatterBotCorpusTrainer, ListTrainer
+Jokes_Path = os.path.expanduser('lib/John/jokes.json')
```

Comment:
By convention Python constants should be in capital letters and words should be separated by underscores:
https://www.python.org/dev/peps/pep-0008/#constants